g3doc/tutorials/graph_keras_cnn_flowers.ipynb

```diff
@@ -0,0 +1,1198 @@
+      "cell_type": "code",
+      "metadata": {
+        "id": "84ffIFmd7PMz"
+      },
+      "source": [
+        "graph_reg_history_dict = graph_reg_history.history\n",
+        "\n",
+        "acc = graph_reg_history_dict['accuracy']\n",
+        "val_acc = graph_reg_history_dict['val_accuracy']\n",
+        "loss = graph_reg_history_dict['loss']\n",
+        "graph_loss = graph_reg_history_dict['scaled_graph_loss']\n",
+        "val_loss = graph_reg_history_dict['val_loss']\n",
+        "\n",
+        "epochs = range(1, len(acc) + 1)\n",
+        "\n",
+        "plt.clf()   # clear figure\n",
+        "\n",
+        "# \"-r^\" is for solid red line with triangle markers.\n",
+        "plt.plot(epochs, loss, '-r^', label='Training loss')\n",
+        "# \"-gD\" is for solid green line with diamond markers.\n",
+        "plt.plot(epochs, graph_loss, '-gD', label='Training graph loss')\n",
+        "# \"-b0\" is for solid blue line with circle markers.\n",
+        "plt.plot(epochs, val_loss, '-bo', label='Validation loss')\n",
+        "plt.title('Training and validation loss')\n",
+        "plt.xlabel('Epochs')\n",
+        "plt.ylabel('Loss')\n",
+        "plt.legend(loc='best')\n",
+        "\n",
+        "plt.show()"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "5EJUJJja7lEB"
+      },
+      "source": [
+        "plt.clf()   # clear figure\n",
+        "\n",
+        "plt.plot(epochs, acc, '-r^', label='Training acc')\n",
+        "plt.plot(epochs, val_acc, '-bo', label='Validation acc')\n",
+        "plt.title('Training and validation accuracy')\n",
+        "plt.xlabel('Epochs')\n",
+        "plt.ylabel('Accuracy')\n",
+        "plt.legend(loc='best')\n",
+        "\n",
+        "plt.show()"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cShQNXGNjIOs"
+      },
+      "source": [
+        "Based on the performances of the two models (with and without graph regularization) we observe that the model performs better without graph regularization. But an ablation study (on the reduced projection dimensionality) reveals a different trend\n",
+        "\n",
+        "| Embedding <br>Dim \t| With <br>Graph-reg \t| Without<br>Graph-reg \t|\n",
+        "|:---:\t|:---:\t|:---:\t|\n",
+        "| 128 \t| 58.36% \t| 56.55% \t|\n",
+        "| 256 \t| 57.82% \t| 55.45% \t|\n",
+        "| 384 \t| 58% \t| 60.18% \t|\n",
+        "| 512 \t| 57.45% \t| 57.64% \t|\n",
+        "\n",
+        "The table above reports the validation top-1 accuracies with and without graph regularization under different reduced embedding dimensionalities. Notice that after 256-d the performance trade-off reverses. Maybe a bit more experimentation with hyperparameters like `similarity_threshold` and `graph_regularization_multiplier` is needed to fully study this behavior. "
```

Comment:
Do you plan to do more work to understand this? If not, I don't know if we need this statement that 'more experimentation is needed....'